main.tf

```diff
@@ -1,14 +1,19 @@
 data "aws_caller_identity" "current" {}
 data "aws_region" "current" {}
 
-resource "aws_elasticache_cluster" "example" {
-  cluster_id           = "cluster-example"
-  engine               = "redis"
-  engine_version       = ""
-  node_type            = "cache.m3.medium"
-  num_cache_nodes      = 1
-  parameter_group_name = "default.redis3.2"
-  port                 = 6379
+resource "random_id" "id" {
+  byte_length = 4
```

Comment:
Can we bump this up please to maybe 8 bytes? Feels too short :)

Reply:
4 bytes of hex is 8 characters long. Still want more?

Reply:
Yes please! The `id` can be up to 20 characters so it fits - more randomess, less collisions.